cmd/podman/main_remote.go

```diff
@@ -32,8 +32,13 @@ func init() {
 
 func setSyslog() error {
 	// Log to file if not using syslog
-	homeDir := homedir.Get()
-	path := filepath.Join(homeDir, ".config", "containers")
+	homeDir := os.Getenv("XDG_CONFIG_HOME")
```

Comment:
I'd re-use `GetRootlessConfigHomeDir` here. See also, comments on https://github.com/containers/libpod/pull/3466/files#diff-018cbf22e8d21da67e6c3605ddcb80acR63.

Nit: `homeDir` is only used a temporary variable, it can be removed.